plugin.json

```diff
@@ -1,8 +1,8 @@
 {
   "id": "mattermost-autolink",
   "name": "Autolink",
+  "version": "1.0.0",
   "description": "Automatically rewrite text matching a regular expression into a Markdown link.",
-  "version": "0.5.1",
```

Comment:
Very, very nit: Why did you changed the order? The previous order was in line with https://developers.mattermost.com/extend/plugins/manifest-reference/.

Reply:
I'll change it back, was an accident, the dog ate my homework